ostree-fdo-container.sh

```diff
@@ -0,0 +1,881 @@
+greenprint "🧹 Clearing container running env"
+# Remove any status containers if exist
+sudo podman ps -a -q --format "{{.ID}}" | sudo xargs --no-run-if-empty podman rm -f
+# Remove all images
+sudo podman rmi -f -a
+
+##########################################################
+##
+## Prepare FDO AIO container
+##
+##########################################################
+greenprint "🔧 Preparing fdo AIO server"
+sudo mkdir aio
+sudo podman run -v "$PWD"/aio/:/aio:z \
+  "quay.io/fido-fdo/aio:nightly" \
+  aio --directory aio generate-configs-and-keys --contact-hostname "$FDO_SERVER_ADDRESS"
+
+sudo /usr/local/bin/yq -iy '.service_info.diskencryption_clevis |= [{disk_label: "/dev/vda4", reencrypt: true, binding: {pin: "tpm2", config: "{}"}}]' aio/configs/serviceinfo_api_server.yml
+# FDO user does not have password, use ssh key and no sudo password instead
+sudo /usr/local/bin/yq -iy '.service_info.initial_user |= {username: "fdouser", sshkeys: ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCzxo5dEcS+LDK/OFAfHo6740EyoDM8aYaCkBala0FnWfMMTOq7PQe04ahB0eFLS3IlQtK5bpgzxBdFGVqF6uT5z4hhaPjQec0G3+BD5Pxo6V+SxShKZo+ZNGU3HVrF9p2V7QH0YFQj5B8F6AicA3fYh2BVUFECTPuMpy5A52ufWu0r4xOFmbU7SIhRQRAQz2u4yjXqBsrpYptAvyzzoN4gjUhNnwOHSPsvFpWoBFkWmqn0ytgHg3Vv9DlHW+45P02QH1UFedXR2MqLnwRI30qqtaOkVS+9rE/dhnR+XPpHHG+hv2TgMDAuQ3IK7Ab5m/yCbN73cxFifH4LST0vVG3Jx45xn+GTeHHhfkAfBSCtya6191jixbqyovpRunCBKexI5cfRPtWOitM3m7Mq26r7LpobMM+oOLUm4p0KKNIthWcmK9tYwXWSuGGfUQ+Y8gt7E0G06ZGbCPHOrxJ8lYQqXsif04piONPA/c9Hq43O99KPNGShONCS9oPFdOLRT3U= ostree-image-test"]}' aio/configs/serviceinfo_api_server.yml
+# No sudo password required by ansible
+tee /tmp/fdouser > /dev/null << EOF
+fdouser ALL=(ALL) NOPASSWD: ALL
+EOF
+sudo /usr/local/bin/yq -iy '.service_info.files |= [{path: "/etc/sudoers.d/fdouser", source_path: "/tmp/fdouser"}]' aio/configs/serviceinfo_api_server.yml
+
+greenprint "🔧 Prepare fdo AIO manufacturing key and cert"
+DIUN_PUB_KEY_ROOT_CERTS=$(sudo cat aio/keys/diun_cert.pem)
+DIUN_PUB_KEY_HASH=sha256:$(sudo openssl x509 -fingerprint -sha256 -noout -in aio/keys/diun_cert.pem | cut -d"=" -f2 | sed 's/://g')
+
+greenprint "🔧 Starting fdo AIO server"
+sudo podman run -d \
+  --ip "$FDO_SERVER_ADDRESS" \
+  --name fdo-aio \
+  --network edge \
+  -v "$PWD"/aio/:/aio:z \
+  "quay.io/fido-fdo/aio:nightly" \
+  aio --directory aio
+
+# Wait for fdo server to be running
+until [ "$(curl -X POST http://${FDO_SERVER_ADDRESS}:8080/ping)" == "pong" ]; do
+    sleep 1;
+done;
+
+##########################################################
+##
+## Build edge-container image and start it in podman
+##
+##########################################################
+
+# Write a blueprint for ostree image.
+tee "$BLUEPRINT_FILE" > /dev/null << EOF
+name = "container"
+description = "A base rhel-edge container image"
+version = "0.0.1"
+modules = []
+groups = []
+
+[[packages]]
+name = "python3"
+version = "*"
+
+[customizations.kernel]
+name = "kernel-rt"
+
+[[customizations.user]]
```

Comment:
User in commit and container is not recommended since RHEL 8.8 and 9.2. You can configured user in simplified installer blueprint.

Reply:
moved user configuration into edge-simplified-installer blueprint.